up-l2/dispatchers/README.adoc

```diff
@@ -64,16 +64,16 @@ In this section we will elaborate on the requirements of the platform Dispatcher
 
 NOTE: These communication layer requirements are still for point-2-point uE communication to and from a dispatcher
 
-* *MUST* support At-least-once delivery policy, this means that the dispatcher will make every attempt to dispatch the CE to the intended Receiver
+* *MUST* support At-least-once delivery policy, this means that the dispatcher will make every attempt to dispatch the UMessage to the intended Receiver
```

Comment:
I got really hung up on this section a while ago. Maybe others reading the spec can get this but I couldn't.

Would suggest adding some verbiage like:
> * If a transport level protocol being used has an at-least-once delivery policy, this requirement is satisfied for that transport protocol.

Reply:
Maybe something like:  This requirement can be met with a combination of uProtocol layers and the transport protocol used.  

Also, while the uProtocol design and implementation MUST have design elements to do this, there are edge cases such as uProtocol stack or host device unexpected shutting down or crashing without warning that result in data loss.  These are beyond this guarantee.